dist/stat_data.py

```diff
@@ -326,10 +326,16 @@ def __init__(self, name, desc, flags=''):
     CacheStat('cache_eviction_queue_empty', 'eviction server candidate queue empty when topping up'),
     CacheStat('cache_eviction_queue_not_empty', 'eviction server candidate queue not empty when topping up'),
     CacheStat('cache_eviction_server_evicting', 'eviction server evicting pages'),
-    CacheStat('cache_eviction_server_slept', 'eviction server slept, because we did not make progress with eviction'),
+    CacheStat('cache_eviction_server_skip_checkpointing_trees', 'skip trees that are being checkpointed'),
```

Comment:
could we begin the description of these stats with `eviction server`?

Reply:
done